components/input/input.css

```diff
@@ -0,0 +1,41 @@
+.input {
+  position: relative;
+  margin: 2.5rem;
+  width: 50%;
+}
+
+.input > input {
+  width: 25rem;
+  height: 4rem;
+  padding: 0.2em;
+  border-radius: 0.5rem;
+}
+
+.input > div {
+  position: absolute;
+  top: -0.6em;
+  font-size: 0.88rem;
+  left: 1em;
+  background-color: var(--primary-color);
+  font-weight: 500;
+  padding: 0.1rem;
+}
+
```

Comment:
Avoid using child selectors, for better code maintainability.

Reply:
Removed them and added separate classes instead. Thank you.